cfme/intelligence/reports/reports.py

```diff
@@ -297,7 +223,59 @@ def _get_state():
             )
 
 
-class CustomSavedReport(Updateable, Pretty):
+@navigator.register(CustomReport, 'All')
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
+
+    def step(self):
+        from cfme.web_ui.menu import nav
+        nav._nav_to_fn('Cloud Intel', 'Reports')(None)
+        accordion.tree("Reports", "All Reports")
+
+
+@navigator.register(CustomReport, 'New')
+class New(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        cfg_btn("Add a new Report")
+
+
+@navigator.register(CustomReport, 'Details')
+class Details(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        accordion.tree(
+            "Reports", "All Reports", "{} (All EVM Groups)".format(
+                store.current_appliance.get_yaml_config("vmdb")["server"]["company"]
+            ), "Custom",
+            self.obj.menu_name)
+        tabstrip.select_tab("Report Info")
+
+
+@navigator.register(CustomReport, 'Edit')
+class Edit(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        cfg_btn("Edit this Report")
+
+
+@navigator.register(CustomReport, 'Saved')
+class Saved(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        accordion.tree(
+            "Reports", "All Reports", "{} (All EVM Groups)".format(
+                store.current_appliance.get_yaml_config("vmdb")["server"]["company"]
```

Comment:
I wonder if we should changed this into a property available on the appliance itself